content/en/agent/cluster_agent/endpointschecks.md

```diff
@@ -15,17 +17,19 @@ further_reading:
 
 ## How it Works
 
-The [Cluster Check][1] feature provides the ability to autodiscover and perform checks on load-balanced cluster services (eg. Kubernetes services). The Endpoints Checks feature extends this mechanism to monitor any endpoint behind cluster services.
```

Comment:
Cluster Checks above is not capitalized. Cluster Check here is capitalized. Just noting in case above is to be capitalized.